utils/supabase/sync-session.ts

```
@@ -0,0 +1,20 @@
+import { createBrowserClient } from "@/utils/supabase/supabase-browser";
+import { access } from "fs";
+
+export async function syncSupabaseSession() {
+    const supabase = createBrowserClient();
+    let {data: {session},error} =await supabase.auth.getSession();
```

Biome (1.9.4)

[error] This let declares some variables that are only assigned once.

'session' is never reassigned.

'error' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)